work/worker.go

```diff
@@ -289,11 +295,27 @@ func (self *worker) update() {
 	quitByErr := make(chan bool, 1)
 	go self.handleTxsCh(quitByErr)
 
+	// Initialize restart watchdog
+	if self.nodetype == common.CONSENSUSNODE && self.restartFn != nil && self.restartTimeOut > 0 {
+		callback := func() {
+			logger.Warn("Consensus timeout")
+			self.restartFn()
+		}
+		logger.Info("Initialize auto restart watchdog", "timeout", self.restartTimeOut.String())
```

Comment:
```suggestion
		logger.Info("Initialize auto restart watchdog", "timeout", self.restartTimeOut)
```
`time.Duration` will be printed as expected without calling `String()`. Could you please check?

Reply:
Good. I reflected. I checked the same result without "String()".